readme.md

```diff
@@ -0,0 +1,237 @@
+# meta-xt-prod-devel
+
+## Content
+This document is long enough to has it's own content. So below you will find:
+* What is meta-xt-prod-devel
+* Quick start
+  * Some basic and most used commands
+* Usage and setup
+  * Serial over USB
+  * SSH over ethernet
+  * What to do "on board"
+* How to build
+  * Few build hints
+* How to install
+  * Write to microSD card
+  * Write to eMMC
+* How to prepare board (loaders and u-boot settings)
```

Comment:
I didn't find this section.

Reply:
Thanks, I have added this section as empty with TODO.